clients/s3.go

```diff
@@ -12,19 +12,19 @@ type S3 interface {
 }
 
 type S3Client struct {
-	s3 *s3.S3
+	S3 *s3.S3
 }
 
 func (c *S3Client) PresignS3(bucket, key string) (string, error) {
-	req, _ := c.s3.GetObjectRequest(&s3.GetObjectInput{
+	req, _ := c.S3.GetObjectRequest(&s3.GetObjectInput{
 		Bucket: &bucket,
 		Key:    &key,
 	})
-	return req.Presign(5 * time.Minute)
+	return req.Presign(60 * time.Minute)
```

Comment:
Why the increase to 60? Also, maybe better to define as a const somewhere common since I see the 60min reference in other places in the code.

Reply:
I think I changed that when I thought we would be sending the presigned urls to mist and didn't know how long it would take mist to process. But since that's not happening I'll reduce it, I think 10 mins makes sense in case there are multiple retries for the probing etc.